wandb/sdk/wandb_init.py

```diff
@@ -345,44 +345,74 @@ def getcaller():
 
 def init(
     job_type: Optional[str] = None,
-    dir=None,
+    dir: Optional[str] = None,
     config: Union[Dict, None] = None,  # TODO(jhr): type is a union for argparse/absl
     project: Optional[str] = None,
     entity: Optional[str] = None,
     reinit: bool = None,
     tags: Optional[List] = None,
-    team: Optional[str] = None,
+    team: Optional[str] = None,  # LB What is this?
     group: Optional[str] = None,
     name: Optional[str] = None,
     notes: Optional[str] = None,
     magic: bool = None,  # TODO(jhr): type is union
-    config_exclude_keys=None,
-    config_include_keys=None,
+    config_exclude_keys: Optional[str] = None,  # LB: Seems unsupported
+    config_include_keys: Optional[str] = None,  # LB: Seems unsupported
     anonymous: Optional[str] = None,
-    disable: bool = None,
-    offline: bool = None,
-    allow_val_change: bool = None,
-    resume=None,
-    force=None,
-    tensorboard=None,  # alias for sync_tensorboard
-    sync_tensorboard=None,
-    monitor_gym=None,
-    id=None,
-    settings: Union[Settings, Dict[str, Any], str, None] = None,
-) -> Run:
-    """Initialize a wandb Run.
+    disable: bool = None,  # LB: What is this
+    offline: bool = None,  # LB: What is this
+    allow_val_change: bool = None,  # LB: What is this?
+    resume: Union[bool, str, None] = None,
+    force: bool = None,
+    tensorboard: bool = None,  # alias for sync_tensorboard, deprecated
+    sync_tensorboard: bool = None,
+    monitor_gym: bool = None,  # LB: Currently unsupported
+    id: Optional[str] = None,
+    settings: Union[Settings, Dict[str, Any], str, None] = None,  # LB: What is this?
```

Comment:
This is a new `wandb.Settings` object that allows configuring all of these settings outside of calls to `wandb.init`.  See https://github.com/wandb/client-ng/blob/master/wandb/sdk/wandb_settings.py#L193 another great candidate for a better doc string.

Reply:
And on the docs page we really need to ducument classes not just methods
Config,Settings,Run,History,Summary